UI/GameSettingsScreen.cpp

```diff
@@ -1158,7 +1166,16 @@ UI::EventReturn GameSettingsScreen::OnJitAffectingSetting(UI::EventParams &e) {
 }
 
 UI::EventReturn GameSettingsScreen::OnChangeMemStickDir(UI::EventParams &e) {
+	#if PPSSPP_PLATFORM(MAC) || PPSSPP_PLATFORM(IOS)
```

Comment:
nit: Shouldn't indent this line or the related below.

Also some mixed tabs/spaces.

-[Unknown]